.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: required
```

Comment:
Yes, I don't see any problem with doing the appropriate user / group configuration.